rpm/sailfish-browser.spec

```diff
@@ -22,6 +22,8 @@ BuildRequires:  qt5-qttools
 BuildRequires:  qt5-qttools-linguist
 BuildRequires:  gdb
 BuildRequires:  oneshot
+BuildRequires:  gtest-devel
+BuildRequires:  libgmock-devel
```

Comment:
Why?
BuildRequires describes source package, not binaries.
Also rpm handles shared lib deps automatically.